ci/prow/config.yaml

```diff
@@ -246,6 +246,7 @@ presubmits:
         - "--artifacts=$(ARTIFACTS)"
         - "--cov-threshold-percentage=80"
         - "--github-token=/etc/covbot-token/token"
+        - "--posting-robot=knative-metrics-robot"
```

Comment:
why are we adding this to the jobs, if its the default value? We should only be adding it if it deviates from default